doc/api/fs.md

```diff
@@ -659,6 +659,8 @@ exist. `data` can be a string or a {Buffer}.
 
 If `options` is a string, then it specifies the `encoding`.
 
+The `mode` option only affects new created file. Same as [`fs.open()`][].
```

Comment:
```suggestion
The `mode` option only affects the new created file.
Same as [`fs.open()`][].
```

Reply:
Rather than "Same as..." I think it reads better like..

```
The `mode` option only affects the newly created file. See [`fs.open()`][]
for more detail.
```